include/deal.II/lac/linear_operator.h

```diff
@@ -1118,6 +1371,227 @@ linear_operator(const OperatorExemplar &operator_exemplar, const Matrix &matrix)
 
 //@}
 
+namespace SchurComplement
+{
+  /**
+   * @name Schur Complement operations
+   */
+  //@{
+
+  /**
+   * @relates LinearOperator
+   *
+   * For the system of equations
+   * @f{eqnarray*}
+       (1) \quad Ax + By &=& f \\
+       (2) \quad Cx + Dy &=& g \quad ,
+   * @f}
+   * this function performs the pre-processing (condensation)
+   * step on the RHS subvector @p g so that the Schur complement
+   * can be used to solve this system of equations.
+   * More specifically, the forward operation modifies the
+   * subvector @p g such that
+   * @f[
+       g \quad \rightarrow \quad g - C \: A^{-1} \: f
+     @f]
+   * The reverse operation, only used in the post-processing
+   * step, reverses this step by modifying the subvector @p g
+   * such that
+   * @f[
+       g \quad \rightarrow \quad g + C \: A^{-1} \: f
+     @f]
+   *
+   * @see
+   * @ref GlossBlockLA "Block (linear algebra)"
+   * @author Jean-Paul Pelteret, 2015
+   *
+   * @ingroup LAOperators
+   */
+  template <typename Range_1, typename Domain_1,
+            typename Range_2>
+  void
+  preprocess (const LinearOperator<Range_1, Domain_1> &A_inv,
+              const LinearOperator<Range_2, Domain_1> &C,
+              const Range_1                           &f,
+              Range_2                                 &g,
+              const bool                               forward_op = true)
```

Comment:
@tamiko Well, thats what you're here for: To remind me that I've forgotten about a useful part of the library that I haven't yet touched :-) Yes, wrapping this in a `PackagedOperation` makes perfect sense. I will follow through!